discovery-provider/plugins/pedalboard/apps/relay/src/scripts/sandbox.ts

```diff
@@ -20,9 +20,9 @@ export const main = async () => {
 
   const apiKey = process.env.SANDBOX_API_KEY;
   const apiSecret = process.env.SANDBOX_API_SECRET;
-  const initialSelectedNode = stagingConfig.discoveryNodes[0]
+  const initialSelectedNode = stagingConfig.discoveryNodes[0];
```

Comment:
is it me or does the linting seems inconsistent throughout this directory? some lines end in semi colons and some don't

Reply:
ah this is probably because i'm not formatting on every commit